.rubocop.yml

```diff
@@ -1,12 +1,28 @@
+AllCops:
+  TargetRubyVersion: 2.3
+  Exclude:
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'spec/**/*'
+
 Documentation:
   Enabled: false
 
+Rails:
+  Enabled: true
+
+# There is a bug in the Rubocop 0.48 parser that causes it to incorrectly
+# complain about parenthese around blocks used with scopes. This should be
+# removed once it is possible to upgrade to 0.49 or above.
```

Comment:
I'm afraid that 0.48 is the latest version of Rubocop that Code Climate supports and I had to manually set an engine in .codeclimate just to be able to use that version.